selfdrive/athena/athenad.py

```diff
@@ -36,11 +36,11 @@
 from openpilot.common.params import Params
 from openpilot.common.realtime import set_core_affinity
 from openpilot.system.hardware import HARDWARE, PC, AGNOS
-from openpilot.system.loggerd.config import ROOT
 from openpilot.system.loggerd.xattr_cache import getxattr, setxattr
 from openpilot.selfdrive.statsd import STATS_DIR
-from openpilot.system.swaglog import SWAGLOG_DIR, cloudlog
+from openpilot.system.swaglog import cloudlog
 from openpilot.system.version import get_commit, get_origin, get_short_branch, get_version
+from selfdrive.hardware.hw import Paths
```

Comment:
shouldn't this be `openpilot.system.hardware.hw`?

Reply:
Yes, both work currently but we want everything to use the full path. I'll look and see if there is a linter rule for it

Reply:
isn't `system` preferred too instead of the symlink?